hivemq4/k8s-image/Dockerfile

```diff
@@ -37,6 +38,8 @@ COPY scripts/*.sh /opt/hivemq/bin/
 COPY --from=unpack /opt/hivemq/extensions/ /opt/hivemq/extensions/
 RUN chmod +x /opt/hivemq/bin/*.sh
 
+RUN apt-get update && apt-get install -y --no-install-recommends unzip && rm -rf /var/lib/apt/lists/*
```

Comment:
Why are we installing unzip? Did the image have it before?

Reply:
The k8s-image depends on the dns-image, which depends on the base-image.
Before this change, the base-image installed the unzip package, so it was available to all (base, dns, k8s).
Only the legacy k8s operator requires unzip, and it uses the k8s-image.